docs/api/archive-ingestion.md

```diff
@@ -32,3 +33,13 @@ import ApiIntro from '../reuse/api-intro.md';
 | KR         | https://api.kr.sumologic.com/docs/#tag/archiveManagement  |
 | US1        | https://api.sumologic.com/docs/#tag/archiveManagement     |
 | US2        | https://api.us2.sumologic.com/docs/#tag/archiveManagement |
+
+<!-- ## Required role capabilities
+
+<ApiRoles/>
```

Comment:
Could you clarify why the `<ApiRoles/>` component is grayed out throughout this PR?